adapters/libuv.h

```diff
@@ -90,7 +90,14 @@ static void on_handle_close(uv_handle_t *handle) {
     // else, wait for `on_timer_close`
 }
 
+#if UV_MAJOR_VERSION > 0 || UV_MINOR_VERSION > 11 || (defined(UV_VERSION_PATCH) && UV_VERSION_PATCH > 22)
```

Comment:
How about this instead:

```suggestion
#if !defined(UV_VERSION_PATCH) || (UV_MAJOR_VERSION < 1 && UV_MINOR_VERSION < 12 && UV_VERSION_PATCH < 23)
```

Looks like `UV_VERSION_PATCH` was introduced in libuv 0.10.2 so if it's not defined, we want the old prototype.
libuv/libuv@c7b1c53ef1afe7c0fef0822d72b0783368b626de

Reply:
looks cleaner👍